### ISSUE TITLE
short order period

### ISSUE THREAD

**Issue by BluemlJ** [d120/pyBuchaktion#43]
If an order period has 10 or less days in total, an order isn't possible.

**Comment by Xiphoseer** [comment]
Could you please provide more detail on this? I cannot reproduce this behavior at all in my testing instance, and without any additional information or log on what it does or doesn't do (or at least the time it happened to compare with server errors), I can't debug it properly. I do not want to disturb the live system to test this right now. Additionally, @akwick, there is nothing at all in the code which would (intentionally) restrict the length of a timeframe.

**Comment by akwick** [comment]
@BluemlJ was able to confirm the bug on Saturday morning and he found the intermediate solution which is working now.
I was also confused by this bug. Currently, we can not reproduce it in the live system as we have a running order phase and as long as it is active, the described behavior won't be triggered. 

What happens: I have created the ordering phase on Friday evening with Friday as starting time with the 21th of May as ending time. At this point, I haven't verified explicitly that it is possible to order books. On Saturday at 11:07 one student reports on Facebook that he isn't able to order. A bit later @BluemlJ  and I tried to solve the problem.
